config/test/test.config.js

```diff
@@ -86,4 +86,9 @@ module.exports = {
     // Only output log messages on test failure. From:
     // https://github.com/facebook/jest/issues/4156#issuecomment-490764080
     // reporters: ["<rootDir>/config/test/log-on-fail-reporter.js"],
+
+    // We turn off Prettier as Prettier v3 is incompatible with Jest v29.
+    // Once they release Jest v30 we can switch to that:
+    // https://github.com/jestjs/jest/issues/14305
```

Comment:
Is there a Jira issue for this Jest upgrade? We should probably link to it here in a TODO.

Reply:
Added: https://khanacademy.atlassian.net/browse/LEMS-2858